glamox/src/Components/BookCard.jsx

```diff
@@ -1,52 +0,0 @@
-import React, { useState } from "react";
```

Comment:
It is recommended that if you are using typescript, then use typescript for the entire project. Don't use it with the mixture of javascript.